plugins/BEdita/Core/tests/TestCase/Model/Table/UsersTableTest.php

```diff
@@ -422,8 +437,12 @@ public function validationSignupProvider()
      * @covers ::validationSignup()
      * @dataProvider validationSignupProvider
      */
-    public function testValidationSignup($expected, array $data)
+    public function testValidationSignup($expected, array $data, array $config = [])
     {
+        Configure::write('Signup', []);
+        if ($config) {
+            Configure::write('Signup', $config);
+        }
```

Comment:
No need for this `if` construct: `Configure::write('Signup', $config);` is enough thanks to the parameter's default value. 🙃